pkg/client/docs/sdks/v3/README.md

```
@@ -2844,6 +2845,59 @@ func main() {
 
+## UpdatePoolQuery
+
+Update the query of a pool
+
+### Example Usage
+
+```go
+package main
+
+import(
+	"context"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)